<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите матрицу ab</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>641</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите данные матрицы отделив элементы друг от друга пробелом,
 а переход на новую строку с помощью ; Например: 1 2 3; 4 5 6 –
это матрица вида:
1 2 3
4 5 6
Для линейного уравнения вида ax = b, где матрица a хранится в ab:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>410</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X равно:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tb1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>571</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ready">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>240</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите матрицу b</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите ненулевые диагонали через пробел</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
